### ISSUE TITLE
Type Error: Cannot read property 'prototype' of undefined

### ISSUE THREAD

**Issue by terminalcommand** [santinic/how2#89]
I'm using arch linux, just installed how2 via npm.  I'm using npm version 6.0.0. I've installed how2@1.5.0.

Whenever, I try to use it, I get the following error:

```
$ how2 -l swift reverse a list
/usr/lib/node_modules/how2/node_modules/devnull/transports/transport.js:59
Transport.prototype.__proto__ = EventEmitter.prototype;
                                             ^

TypeError: Cannot read property 'prototype' of undefined
    at Object.<anonymous> (/usr/lib/node_modules/how2/node_modules/devnull/transports/transport.js:59:46)
    at Module._compile (internal/modules/cjs/loader.js:678:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)
    at Module.load (internal/modules/cjs/loader.js:589:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)
    at Function.Module._load (internal/modules/cjs/loader.js:520:3)
    at Module.require (internal/modules/cjs/loader.js:626:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/usr/lib/node_modules/how2/node_modules/devnull/transports/stream.js:8:17)
    at Module._compile (internal/modules/cjs/loader.js:678:30)
````

**Comment by PaoloCifariello** [comment]
Hi @terminalcommand, thank you for the report.
This issue was discussed in #79 and is currently solved on GitHub. 
In the README we still point to `how2` NPM repo, while the updated one is `how-2`. 

TL;DR;
```
npm uninstall -g how2
npm i -g how-2
```

**Comment by terminalcommand** [comment]
Thank you for the super quick response! Much appreciated.
